File: 백준/Bronze/16673. 고려대학교에는 공식 와인이 있다/고려대학교에는 공식 와인이 있다.cc
#include<stdio.h>

int main() {
    int C, K, P, count=0;
    scanf("%d %d %d", &C, &K, &P);
    
    for (int i=1; i < C + 1; i++){
        count += K * i + P * (i * i);
    }
    
    printf("%d", count);
}

File: 백준/Bronze/16673. 고려대학교에는 공식 와인이 있다/고려대학교에는 공식 와인이 있다.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
      int C, K, P, count=0;
      Scanner s = new Scanner(System.in);
      C = s.nextInt();
      K = s.nextInt();
      P = s.nextInt();
    
    for (int i=1; i < C + 1; i++){
        count += K * i + P * (i * i);
    }
      
      System.out.print(count);
    }
}